tests/copp/test_copp.py

```diff
@@ -148,7 +163,7 @@ def _copp_runner(dut, ptf, protocol, test_params):
                testdir="ptftests",
                # Special Handling for DHCP if we are using T1 Topo
                testname="copp_tests.{}Test".format((protocol+"TopoT1")
-                         if test_params.topo in _SUPPORTED_T1_TOPOS and protocol in _T1_NO_COPP_PROTOCOL else protocol),
+                         if protocol in _TOR_ONLY_PROTOCOL and dut_type != "ToRRouter" else protocol),
```

Comment:
T1 is still missing ? I thought you updated.

Reply:
It's not missing. The check is made for ToR DUT type taken from DEVICE_METADATA and if it's "ToRRouter" the regular DHCPTest is run. If not - the DHCPTopoT1Test. And this is done because on ptf32 topology I'm running the test with the device type is "LeafRouter" so if leave the check as is the DHCPTest will be executed and will fail as there is no DHCP trap group configured.

Reply:
@vsenchyshyn thanks.. 

@daall for PTF-32 what can be device type ? It is always Leaf or it can be ToR also ?

Reply:
ptf32 and ptf64 both use t1 and t1-64 respectively, so it should always be LeafRouter.

Reply:
@abdosi Do you have any concerns with the changes? If not, could you please approve and merge?